app/application.py

```diff
@@ -87,3 +66,28 @@ def handle_save_exception(error):
     response = jsonify(error.to_dict())
     response.status_code = error.status_code
     return response
+
+
+def log_request():
+    header = request.headers
```

Comment:
Add a description e.g     
"""Line 1 ...
    Line 2 of comment...
    And so on... 
    """